definitions/divorce/json/CaseField/CaseField-payment-by-account.json

```diff
@@ -0,0 +1,10 @@
+[
```

Comment:
this file should have "-nonprod" or "-prod" postfix.

Reply:
Yeah youre right. Ill update now